documentation/library_guide/parallel_api/pass_data_algorithms.rst

```diff
@@ -181,3 +183,48 @@ combination with ``std::vector::size()`` as shown in the example above, rather t
 be possible for |onedpl_short| to detect that iterators are pointing to USM-allocated data. In that
 case the data will be treated as if it were host-allocated, with an extra copy made to a SYCL buffer.
 Retrieving USM pointers from ``std::vector`` as shown guarantees no unintended copying.
+
+Use Range Views
+---------------
+
+For :doc:`parallel range algorithms <parallel_range_algorithms>' with device execution policies,
```

Comment:
There are a couple of rendering issues. For example, this line has a typo (' instead of `).
 
[doc_fix.patch](https://github.com/user-attachments/files/17017940/doc_fix.patch)